cmd/influx/main.go

```diff
@@ -134,24 +134,57 @@ func out(w io.Writer) genericCLIOptFn {
 	}
 }
 
-func err(w io.Writer) genericCLIOptFn {
-	return func(o *genericCLIOpts) {
-		o.errW = w
-	}
-}
-
-func runEMiddlware(mw cobraRunEMiddleware) genericCLIOptFn {
-	return func(o *genericCLIOpts) {
-		o.runEWrapFn = mw
-	}
-}
-
 type globalFlags struct {
 	config.Config
 	skipVerify   bool
 	traceDebugID string
 }
 
+func (g *globalFlags) registerFlags(cmd *cobra.Command, skipFlags ...string) {
+	if g == nil {
+		panic("global flags are not set: <nil>")
+	}
+
+	skips := make(map[string]bool)
+	for _, flag := range skipFlags {
+		skips[flag] = true
+	}
+
+	fOpts := flagOpts{
+		{
+			DestP: &g.Token,
+			Flag:  "token",
+			Short: 't',
+			Desc:  "Authentication token to be used throughout client calls",
```

Comment:
Could this just be "Authentication token"?

~to be used throughout client calls~